Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -260,13 +260,18 @@ def _get_itk_pixelid(numpy_array_type):
     """Returns a ITK PixelID given a numpy array."""
 
     import itk
+    def _long_type():
+        if os.name == "nt":
+            return itk.ULL
+        else:
+            return itk.UL
```

Comment:
Looking good! I wasn't sure about using `os.name` here at first, but on investigation we are doing the exact same platform check to define the width of `itk.UL` in [`types.py`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itk/support/types.py#L125). So, carry on 😄

Reply:
Yes, that's what we are following everywhere.

Reply:
Changed the pathname. I tested it locally with a different vtk file.